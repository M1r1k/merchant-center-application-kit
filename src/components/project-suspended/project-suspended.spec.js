import React from 'react';
import { shallow } from 'enzyme';
import ProjectSuspended from './project-suspended';

describe('rendering', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ProjectSuspended />);
  });
  it('outputs correct tree', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render layout component', () => {
    expect(wrapper).toRender('ServicePageResponseLayout');
  });
});
