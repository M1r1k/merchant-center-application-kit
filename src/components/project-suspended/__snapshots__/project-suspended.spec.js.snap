// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering outputs correct tree 1`] = `
<ServicePageResponseLayout
  bodyContent={<withRouter(ServicePageProjectSwitcher) />}
  imageSrc="test-file-stub"
  paragraph1={
    <FormattedMessage
      defaultMessage="Please contact your system administrator if you have any further questions or select another Project to view."
      id="ProjectSuspended.paragraph1"
      values={Object {}}
    />
  }
  title={
    <FormattedMessage
      defaultMessage="Your Project has been suspended"
      id="ProjectSuspended.title"
      values={Object {}}
    />
  }
/>
`;
